### PR TITLE
Adding redux-cli custom blueprints

### PR DESCRIPTION
**Pull request by vittorius**

### CODE REVIEW

File: blueprints/smart/files/__root__/__smart__/__name__.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class <%= pascalEntityName %> extends Component {
+  render() {
+    return (
+      <div></div>
+    );
+  }
+}
+
+const mapStateToProps = (state) => {
```
Comment (roman-holovin) [inline blueprints/smart/files/__root__/__smart__/__name__.js]: It is more convenient to use short forms of the map functions. Also, these should be placed at the top the file. This way you can see data flow before component code that uses this data.

File: blueprints/dumb/files/__root__/__dumb__/__name__.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { Component, PropTypes } from 'react'
```
Comment (roman-holovin) [inline blueprints/dumb/files/__root__/__dumb__/__name__.js]: Starting with react 15.5.5 `PropTypes` are now separate module, this code will issue a warning.
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes

File: blueprints/duck/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const entityNameLocals = require('../util').entityNameLocals;
```
Comment (roman-holovin) [inline blueprints/duck/index.js]: Either you place semicolons everywhere or omit them entirely.
You can define eslint rule about semicolons and use `eslint ---fix` to make it consistent everywhere

File: blueprints/duck/index.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+const separator = require('path').sep;
+
+module.exports = {
+  locals(options) {
```
Comment (roman-holovin) [inline blueprints/duck/index.js]: This function is redundant.

File: blueprints/duck/index.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    return entityNameLocals(options);
+  },
+  description() {
+    return 'Generates a reducer/actions/constant Duck Module for redux';
+  },
+  fileMapTokens() {
+    return {
+      __duck__: (options) => {
+        return `redux/modules${separator}${options.locals.entityPath}`;
```
Comment (roman-holovin) [inline blueprints/duck/index.js]: This `return`is redundant

File: blueprints/form/files/__root__/__form__/__name__.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      </form>
+    )
+  }
+}
+
+<%= pascalEntityName %>.propTypes = propTypes
+<%= pascalEntityName %> = reduxForm({
+  form: '<%= pascalEntityName %>',
+  fields,
+  validate
```
Comment (roman-holovin) [inline blueprints/form/files/__root__/__form__/__name__.js]: Usage of trailing comma is inconsistent.

File: blueprints/util.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    baseName
+  };
+}
+
+module.exports = {
+  entityNameLocals(blueprintOptions) {
+    const epInfo = entityPathInfo(blueprintOptions.entity.name);
+
+    const entityNames = ['pascal', 'camel', 'snake', 'dashes'].reduce(
+      (locals, variant) => Object.assign(locals, { [`${variant}EntityName`]: normalizeCasing(epInfo.baseName, variant)}),
```
Comment (roman-holovin) [inline blueprints/util.js]: Inconsistent usage of spacing in object declaration

**Review by roman-holovin — CHANGES_REQUESTED** [review]
My general recommendation will be eslint configuration. Default config from CRA are too lax. `eslint-config-airbnb` is more strict and has many additional rules enabled. This will ensure code style consistency.

File: blueprints/form/files/__root__/__form__/__name__.js (hunk 6 above)
Comment (vittorius) [inline blueprints/form/files/__root__/__form__/__name__.js]: Made consistent with ESLint config.

**Review by roman-holovin — APPROVED** [review]

**Pull request merged**